installing/installing_aws/installing-aws-user-infra.adoc

```diff
@@ -114,6 +114,8 @@ include::modules/installation-operators-config.adoc[leveloffset=+1]
 
 include::modules/installation-registry-storage-config.adoc[leveloffset=+2]
 
+You can configure registry storage for user-provisioned infrastructure in AWS to deploy {product-title} to hidden regions. See xref:../../registry/configuring_registry_storage/configuring-registry-storage-aws-user-infrastructure.adoc#configuring-registry-storage-aws-user-infrastructure[Configuring the registry for AWS user-provisioned infrastructure] for more information.
```

Comment:
I've confused myself trying to find the right preview build page. The one I came to was this: https://osdocs-1159-s3--ocpdocs.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-user-infra.html#installation-registry-storage-config_installing-aws-user-infra but it is slightly different from your text. The preview has "...to deploy OCP to _AWS GovCloud and_ hidden regions." Calling this out in case something isn't right?